subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/precompile/v1/PrecompiledScriptTemplates.kt

```diff
@@ -201,6 +202,7 @@ open class PrecompiledProjectScript(
 
 internal
 object PrecompiledInitScriptCompilationConfiguration : ScriptCompilationConfiguration({
+    isStandalone(false)
```

Comment:
This is an interesting change. Do you have more information about the "warning emitted by KotlinCompile tasks", @eskatos? What's the exact wording of that warning? And what type of scripts need to set this, i.e. what defines "standalone"?

Reply:
A standalone script is a script not in a source set.
Here's the related issue https://youtrack.jetbrains.com/issue/KT-54325
And the exact warning:
```
w: Script 'something.gradle.kts' is not supposed to be used along with regular Kotlin sources, and will be ignored in the future versions by default. (Use -Xallow-any-scripts-in-source-roots command line option to opt-in for the old behavior.)
```